helpers/class.metrilo-api-client.php

```diff
@@ -6,16 +6,15 @@
 
 class Metrilo_Api_Client {
     
-    public function get_client()
+    public function get_client($api_end_point)
```

Comment:
Lets call this something with `domain`, like the `$tracking_endpoint_domain` variable.
I would suggest `$api_endpoint_domain` as an obvious candidate.

Reply:
Adopting your suggestion.